lib/competition_api.rb

```diff
@@ -12,7 +12,7 @@ def self.find(competition_id)
     nil
   end
 
-  def self.comp_api_url(competition_id)
+  def self.url(competition_id)
```

Comment:
Why this rename?

Reply:
CompetitionApi.comp_api_url seems redundant - CompetitionApi.url communicates basically the same thing